Week3/script.js

```diff
@@ -135,7 +134,7 @@ vehicle("blue",1,5);
 
         // 7
 
-let vehicles = ["motorbike", "caravan", "bike", "car", ]
+let vehicles = [" motorbike", " caravan", " bike", " car"];
```

Comment:
Ah, why did you add the spaces here? I think it was fine before (except for the `,`)